internal/cmd/root.go:17:26: too many arguments in call to gap.NewScope

after downloading qamel,run `go get -u -v github.com /go-qamel/qamel/cmd/qamel`,

throw an error:

internal/cmd/root.go:17:26: too many arguments in call to gap.NewScope
     have (gap.ScopeType, string, string)
     want (gap.ScopeType,string)

OS: Manjaro 19.0.2 Kyria
Kernel: x86_64 Linux 5.5.8-1-MANJARO

--- a/go.mod
+++ b/go.mod
@@ -6,7 +6,7 @@ require (
 	github.com/fatih/color v1.7.0
 	github.com/fsnotify/fsnotify v1.4.7
 	github.com/mattn/go-colorable v0.1.2 // indirect
-	github.com/muesli/go-app-paths v0.0.0-20181030220709-913f7f7ac60f
+	github.com/muesli/go-app-paths v0.2.1
 	github.com/sirupsen/logrus v1.4.2
 	github.com/spf13/cobra v0.0.5
 )

--- a/internal/cmd/root.go
+++ b/internal/cmd/root.go
@@ -14,7 +14,7 @@ var (
 func init() {
 	// Get config path in ${XDG_CONFIG_HOME}/qamel/config.json
 	var err error
-	userScope := ap.NewScope(ap.User, "qamel", "qamel")
+	userScope := ap.NewScope(ap.User, "qamel")
 	configPath, err = userScope.ConfigPath("config.json")
 	if err != nil {
 		log.Fatalln(err)

